src/db/z_db.erl

```diff
@@ -417,7 +429,7 @@ insert(Table, Props, Context) ->
 
 
 %% @doc Update a row in a table, merging the props list with any new props values
-%% @spec update(Table, Id, Parameters, Context) -> {ok, RowsUpdated}
+-spec update(table_name(), id(), parameters(), #context{}) -> {ok, integer()}.
```

Comment:
The spec in line 432 uses a record, please define a type for the record and use that instead.